lua/luasnip/extras/snippet_list.lua

```diff
@@ -0,0 +1,92 @@
+local function get_context(snippet)
+	return {
+		name = snippet.name,
+		trigger = snippet.trigger,
+		description = snippet.dscr,
+		wordTrig = snippet.wordTrig and true or false,
+		regTrig = snippet.regTrig and true or false,
+		docstring = snippet:get_docstring(),
+	}
+end
+
+local function get_name(bufnr)
+	return "LuaSnip://Snippets buf " .. bufnr
+end
+
+local function printer(snippets)
+	return vim.inspect(snippets)
+end
```

Comment:
I don't think we have to wrap `vim.inspect` here, we could just `opts.printer = opts.printer or printer`.
(alternatively, `local printer = vim.inspect`)

Reply:
Solved in https://github.com/L3MON4D3/LuaSnip/pull/652/commits/5d2cd4505e27b19af24a6156a4daadcc28885c07